samples/scenarios/snowflake_oauth/oauthConfig.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<pluginOAuthConfig>	 
+	<dbclass>snowflake_oauth</dbclass>
+    <clientIdDesktop>UP+iCpaEoSG9qM1Q1+Xx6kpxwF0=</clientIdDesktop>
+    <clientSecretDesktop>EAofGQyN1du/vqpIK/eThPGhLNBHNyb9DA9Js8bk2YA=</clientSecretDesktop>
```

Comment:
Are we comfortable with these being out in the open? If there's any doubt, we should just replace these fields with XXXXXXX, the connector doesn't technically need to work, just serve as an example.

Reply:
Will mask them